src/Compilers/Core/Portable/Syntax/SyntaxNode.cs

```diff
@@ -1403,10 +1405,26 @@ public virtual void SerializeTo(Stream stream, CancellationToken cancellationTok
                 throw new InvalidOperationException(CodeAnalysisResources.TheStreamCannotBeWrittenTo);
             }
 
+            // Report NFW to see if this is being used in the wild.
+            FatalError.ReportAndPropagate(new SerializationDeprecationException());
```

Comment:
Same as other place.

Reply:
This is the check that is causing the MVID test to fo fail.